tests/st/utils/data.py

```diff
@@ -217,23 +213,19 @@
                     'notNets': ['10.3.0.0/16'],
                     'notPorts': ['110:1050'],
                     'notSelector': "type=='apples'",
-                    'notTag': "bananas",
```

Comment:
💯

Reply:
hmmm, are we not testing with this resource def?  If not why is this not failing tests?

Reply:
oh lordy I misread this.  Ignore me